src/machine.tex

```diff
@@ -1364,17 +1364,22 @@ \subsection{Machine Interrupt Registers ({\tt mip} and {\tt mie})}
 \label{miereg}
 \end{figure}
 
-An interrupt \textit{i} will be taken if bit \textit{i} is set in both
-{\tt mip} and {\tt mie}, and if interrupts are globally enabled.  By
-default, M-mode interrupts are globally enabled if the hart's current
-privilege mode is less than M, or if the current privilege mode is M
-and the MIE bit in the {\tt mstatus} register is set.  If bit \textit{i}
-in {\tt mideleg} is set, however, interrupts are considered to be
-globally enabled if the hart's current privilege mode equals the
-delegated privilege mode and that mode's interrupt enable
-bit (\textit{x}\/IE in {\tt mstatus} for mode~\textit{x}) is set,
-or if the current
-privilege mode is less than the delegated privilege mode.
+An interrupt~\textit{i} will trap to M-mode (causing the privilege mode
+to change to M-mode) if all of the following are true:
```

Comment:
The preceding parenthetical comment is inconsistent in that there isn't a similar parenthetical comment for an interrupt trapping to HS-mode.  And, like above for HS-mode, this doesn't seem necessary to say (or otherwise needs to be said in both places).

Reply:
IIRC, @jhauser-us excluded the corresponding remark from the hypervisor chapter because it's redundant with the one in the supervisor chapter.